conan_provider.cmake

```diff
@@ -110,6 +154,13 @@ function(detect_host_profile output_file)
 
     string(APPEND PROFILE "[conf]\n")
     string(APPEND PROFILE "tools.cmake.cmaketoolchain:generator=${CMAKE_GENERATOR}\n")
+    if(${MYOS} STREQUAL "Android")
+        if(EXISTS $ENV{ANDROID_NDK_ROOT})
+            string(APPEND PROFILE "tools.android:ndk_path={{ os.getenv('ANDROID_NDK_ROOT') }}\n")
```

Comment:
Does _using_ the Android NDK _require_ this path to be defined as an environment variable when invoking CMake - or can cmake be invoked simply by passing the absolute path to the android toolchain file?

Because I would expect that at this stage, if the Android toolchain has already been processed, the following variable:
https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_NDK.html points to the correct path - which would mean an env var is not strictly required.

Reply:
No, it doesn't, I should have dug deeper. I switched to using the `CMAKE_ANDROID_NDK` variable instead. That variable gets set by the Android toolchain and it's one of the [documented ways](https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-for-android-with-the-ndk) to define the NDK path without the toolchain.